docker/java/spring/maven-package.sh

```diff
@@ -1,25 +1,29 @@
 #!/usr/bin/env bash
-set -x
+set -xe
 JAVA_AGENT_BUILT_VERSION=${1}
 
 ARTIFACT_ID=elastic-apm-agent
 
 if [ -z "${JAVA_AGENT_BUILT_VERSION}" ] ; then
   cd /agent/apm-agent-java
-  mvn -q --batch-mode install -DskipTests \
+  git log -1
```

Comment:
show the last commit in the source, useful to know if you are compiling what you want